framework/composer.json

```diff
@@ -54,7 +54,6 @@
     "require": {
         "php": ">=5.4.0",
         "ext-mbstring": "*",
-        "ext-mcrypt": "*",
```

Comment:
It does not, necessarily. The only thing that does not work is the composer asset plugin.